### ISSUE TITLE
Spread not implemented yet - stylex.props(...styles)

### ISSUE THREAD

**Issue by BjornHMS** [Dwlad90/stylex-swc-plugin#54]
When attempting to use stylex.props with a spread argument anywhere in the code, it crashes.

```
The application panicked (crashed).
Message:  Spread not implemented yet
Location: crates\stylex-shared\src\shared\utils\core\stylex_merge.rs:34

Run with COLORBT_SHOW_HIDDEN=1 environment variable to disable frame filtering.
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ BACKTRACE ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 1: napi_register_module_v1
    at <unknown source file>
 2: napi_register_module_v1
    at <unknown source file>
 3: napi_register_module_v1
    at <unknown source file>
 4: napi_register_module_v1
    at <unknown source file>
 5: napi_register_module_v1
    at <unknown source file>
 6: napi_register_module_v1
    at <unknown source file>
 7: napi_register_module_v1
    at <unknown source file>
 8: napi_register_module_v1
    at <unknown source file>
 9: napi_register_module_v1
    at <unknown source file>
10: v8::base::CPU::has_idiva
    at <unknown source file>
11: v8::SharedValueConveyor::SharedValueConveyor
    at <unknown source file>
12: v8::SharedValueConveyor::SharedValueConveyor
    at <unknown source file>
13: v8::SharedValueConveyor::SharedValueConveyor
    at <unknown source file>
14: v8::SharedValueConveyor::SharedValueConveyor
    at <unknown source file>
15: v8::PropertyDescriptor::writable
    at <unknown source file>
16: v8::PropertyDescriptor::writable
    at <unknown source file>
17: v8::PropertyDescriptor::writable
    at <unknown source file>
18: v8::PropertyDescriptor::writable
    at <unknown source file>
19: v8::PrimitiveArray::Length
    at <unknown source file>
20: v8::PrimitiveArray::Length
    at <unknown source file>
21: v8::PrimitiveArray::Length
    at <unknown source file>
22: v8::base::CPU::has_sse41
    at <unknown source file>
23: v8::base::CPU::has_sse41
    at <unknown source file>
24: node::CallbackScope::~CallbackScope
    at <unknown source file>
25: node::CallbackScope::~CallbackScope
    at <unknown source file>
26: node_module_register
    at <unknown source file>
27: uv_timer_stop
    at <unknown source file>
28: uv_timer_stop
    at <unknown source file>
29: uv_update_time
    at <unknown source file>
30: uv_run
    at <unknown source file>
31: node::SpinEventLoop
    at <unknown source file>
32: X509_STORE_get_cleanup
    at <unknown source file>
33: node::Start
    at <unknown source file>
34: node::Start
    at <unknown source file>
35: AES_cbc_encrypt
    at <unknown source file>
36: inflateValidate
    at <unknown source file>
37: BaseThreadInitThunk
    at <unknown source file>
38: RtlUserThreadStart
    at <unknown source file>

```

**Comment by Dwlad90** [comment]
Hi @BjornHMS,

Could you provide an example of code that throws the error?

**Comment by BjornHMS** [comment]
@Dwlad90 
```typescript
Const styleArr = [styles.root, styles.stuff];
....
Stylex.props(...styleArr)
```
Basically any use of spread in .props.
Passing the array directly works though, I figured i'd still submit the issue as its throwing errors and crashing.

**Comment by Dwlad90** [comment]
Thanks for the example provided and for helping to develop the project.

Version "0.3.1-rc.2"  with the fix for this issue was uploaded to NPM

**Comment by BjornHMS** [comment]
> The fix is ​​available in the "0.3.1-rc.2" plugin version

No problem @Dwlad90 It's the least I could do, thank you very much for your swift fixes!
The issue has been resolved.
